src/pages/services.js

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import Layout from "../components/Layout"
+import Services from "../components/Services"
+import styles from "../css/services.module.scss"
+import { graphql } from "gatsby"
+
+export default function servicesPage({ data: { design, develop } }) {
+  console.log(design, develop)
+
+  return (
+    <Layout>
+      <h1>Services</h1>
+      <section className={styles.section}>
+        <Services
+          service="Wed-Design"
+          fluid={design.nodes[0].childImageSharp.fluid}
+        />
+        <Services
+          service="Web-Development"
+          fluid={develop.nodes[0].childImageSharp.fluid}
+        />
+      </section>
+    </Layout>
+  )
+}
+
+export const query = graphql`
+  query ServicePageQuery {
```

Comment:
@liuba-larousse what's the error exactly that you are having with gatsby?

Reply:
I started the project and the console shows a detailed log with the errors and some solutions. Maybe it will help you to fix the issue you are having :)

Reply:
The error would say : can not query "fluid" of undefined. And when I logged in the query , nodes would return as an array. Thats why I had to put [0]

Reply:
I can't start the project, it says

```
error There was an error in your GraphQL query:

Cannot query field "allStrapiProject" on type "Query".

If you don't expect "allStrapiProject" to exist on the type "Query" it is most likely a typo.
However, if you expect "allStrapiProject" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "allStrapiProject" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "Query":
https://www.gatsbyjs.org/docs/schema-customization/#creating-type-definitions
error "gatsby-node.js" threw an error while running the createPages lifecycle:
```

Maybe there's something you missed to commit regarding strapi?